components/homeComponents/HomeFooter.tsx

```diff
@@ -29,14 +29,26 @@ export default function HomeFooter() {
         {/* HackUTD Section */}
         <div className="flex-1 p-4">
           <h1 className="font-bold text-2xl">HackUTD</h1>
-          <Link href="mailto:hello@hackutd.co">
-            <p
-              className="underline mt-1 font-medium"
-              style={{ whiteSpace: 'nowrap', wordBreak: 'keep-all', textUnderlineOffset: '2px' }}
+          <p style={{ whiteSpace: 'nowrap', wordBreak: 'keep-all', textUnderlineOffset: '2px' }}>
+            Contact Us
+          </p>
+
+          {/* Social Media Links */}
+          <div className="flex space-x-4 mt-4">
+            <a href="https://instagram.com/hackutd" target="_blank" rel="noopener noreferrer">
+              <img src="/assets/logo-instagram.png" alt="Instagram" className="w-10 h-10" />
+            </a>
+            <a
+              href="https://linkedin.com/company/hackutd"
+              target="_blank"
+              rel="noopener noreferrer"
             >
-              Contact Us
-            </p>
-          </Link>
+              <img src="/assets/logo-linkedin.png" alt="LinkedIn" className="w-10 h-10" />
+            </a>
+            <a href="mailto:hello@hackutd.co">
+              <img src="/assets/email-logo.png" alt="Email" className="w-10 h-10" />
+            </a>
+          </div>
```

Comment:
Logo is a bit blurry in my opinion. Can we use logo from somewhere else like heroicons?